packages/hash/frontend/src/components/hooks/useCreatePage.ts

```diff
@@ -1,24 +1,27 @@
 import { useMutation } from "@apollo/client";
 import { useRouter } from "next/router";
 
-import { createPage } from "@hashintel/hash-shared/queries/page.queries";
+import {
+  createPage,
+  setParentPage,
+} from "@hashintel/hash-shared/queries/page.queries";
+import {
+  SetParentPageMutation,
+  SetParentPageMutationVariables,
+} from "@hashintel/hash-shared/graphql/apiTypes.gen";
 import {
   CreatePageMutation,
   CreatePageMutationVariables,
 } from "../../graphql/apiTypes.gen";
 import { getAccountPages } from "../../graphql/queries/account.queries";
 
-export const useCreatePage = () => {
+export const useCreatePage = (accountId: string) => {
```

Comment:
I agree, but I think that makes when we have multiple parameters. For hooks with just 1 parameter we just pass in the parameter (based on similar hooks in the codebase). I don't see the need to use an object in that scenario

Reply:
I see, I'm realising this is something we're more consistent about in the BE than in the FE.

In which case no need to change it, but in my opinion it would be an additional safety mechanism that helps developers ensure they are passing the correct variable to a function, as they have to be explicit about the variable name when passing it to an object:
```TypeScript
const localSpecialId = "...";

const { } = someHook({ specialID: localSpecialId });
```
but can pass any variable when passing it directly as a function argument:
```TypeScript
const someString = "...";

const { } = someHook(someString);
```
Which is why in my opinion it can more quickly lead to mistakes. But doesn't have to be changed, just wanted to explain my thinking.